source/publishing_data/05_processing_data/processors/expression.rst

```diff
@@ -1,273 +1,347 @@
-   * * Binary operators
-     * +, -, ``*``, /, % (euclidean division), and, &&, or, ||, >, <, >=, <=, ==, != (evaluates to ``True`` or ``False``), &
-       (concatenation of strings, evaluates to a string)
-   * * Ternary operators
-     * op1 ? op2 : op3 (conditional statement). Please note that op3 is optional, so you can write op1 ? op2.
+- **unary operators** can be used as prefixes or suffixes to alter the value of 1 expression,
+- **binary operators** can be arithmetic operators to perform a calculus between 2 expressions, or boolean operators to compare the result of 2 expressions,
+- the **ternary operator**, to convert a conditional expression to either 1 of 2 possible results.
 
-In the table below are listed the available functions:
+Unary operators
+^^^^^^^^^^^^^^^
 
 .. list-table::
    :header-rows: 1
 
-   * * Function type
-     * Functions
-   * * Without operand
-     * now
-   * * Single operand
-     * abs, acos, acosh, asin, asinh, atan, atanh, capitalize, capitalize_all, ceil, cos, day, dayofweek, degrees, empty, exp, factorial, floor, fromtimestamp, gamma, hour, isalnum, isalpha, isdecimal, isdigit, islower, isnumeric, isupper, length, log, log10, lower, minute, month, normalize, quarter, quartertodaterange, radians, random, round, second, sigmoid, sin, sqrt, tan, tolowercase, touppercase, trunc, upper, week, year
-   * * Two operands
-     * add_days, add_hours, add_minutes, add_months, add_seconds, add_years, contains, day, dayofweek, distance, div, endswith, gcd, geopoint2d_towgs84, geoshape_towgs84, hour, match, max, microsecond, min, minute, mod, month, pow, quarter, quartertodaterange, random, round, second, startswith, substring, week, year
-   * * Three operands
-     * center, datediff, ljust, replace, rjust, substring
+   * * Operator
+     * Description
+     * Example
+   * * ``-``
+     * Prefix that negates the following value
+     * ``- 4``, ``- [expression]``
+   * * ``not``, ``!`` (not)
+     * Boolean operator that inverts the following condition
+     * * ``not true`` or  ``!true`` returns ``false``
+       * ``not 4 > 5`` returns ``true``
+       * ``!(5 <= 10)`` returns ``false``
+   * * ``!`` (factorial)
+     * Suffix that computes the factorial of an expression
+     * ``3!`` returns ``1*2*3``
+
+Binary operators
+^^^^^^^^^^^^^^^^
 
+.. list-table::
+   :header-rows: 1
 
-Examples
---------
+   * * Operator
+     * Description
+     * Example
+   * * ``+``, ``-``, ``*``, ``/``, ``%``, ``//``, ``^``
+     * Arithmetic operators: add, subtract, multiply, divide, modulo, euclidian division, power
+     * * ``2 + 3`` returns ``5``
+       * ``5 - 8`` returns ``-3``
+       * ``9 * 3`` returns ``27``
+       * ``15 / 6`` returns ``2.5``
+       * ``15 % 6`` returns ``3``
+       * ``15 // 6`` returns ``2``
+       * ``2 ^ 3`` returns ``8``
+       * ``column_1 + column2``
+   * * ``and``, ``&&``, ``or``, ``||`` (return a boolean)
+     * Boolean operators: and, and, or, or
```

Comment:
A little strange to duplicate this. Don't we want a list instead ? Is it necessary to speak about binary operator priority ? e.g.: if you have a || and a && in the same expression, what is the operator priority.

Reply:
Indeed, I may have talked about operator precedence. Let's add it in another change so this one can get translated and published.

Reply:
Done